Gemfile.lock

```diff
@@ -235,6 +242,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86-mingw32
```

Comment:
Same as the previous PR with these mingw32 dependencies. I wonder if we could automatically filter those out for you...

Reply:
done